packages/installer/upgrade.sh

```diff
@@ -160,19 +160,24 @@ upgrade() {
     export LUET_PRIVILEGED_EXTRACT=true
 
     args=""
-    if [ -z "$VERIFY" ]; then
-        args="--enable-logfile --logfile /tmp/luet.log --plugin image-mtree-check"
+    if [ -z "$VERIFY" ] || [ "$VERIFY" == true ]; then
+        args="--enable-logfile --logfile /tmp/luet.log --plugin luet-mtree"
     fi
+
     if [ -n "$CHANNEL_UPGRADES" ] && [ "$CHANNEL_UPGRADES" == true ]; then
         echo "Upgrading from release channel"
+        set -x
```

Comment:
Is this a leftover from debugging?

Reply:
yes/no.. I've added it for debugging, but I'm unsure to keep it. It seems useful for debug, but also as a informative output of what's going on.. UX definetly deserve its own story :)